packages/web-components/tests/snapshots/dds-region-item.md

```diff
@@ -28,23 +28,24 @@
 ####   `should render with invalid state`
 
 ```
-<a
-  class="bx--link"
-  href="#"
+<p
+  class="bx--link bx--link--disabled"
   id="link"
 >
-  <div class="bx--card__wrapper">
-    <div class="bx--card__content">
-      <h3 class="bx--card__heading">
-        <slot>
-          name-foo
-        </slot>
-      </h3>
-      <div class="bx--card__footer">
-      </div>
+</p>
+<div class="bx--card__wrapper">
+  <div class="bx--card__content">
+    <h3 class="bx--card__heading">
+      <slot>
+        name-foo
+      </slot>
+    </h3>
+    <div class="bx--card__footer">
```

Comment:
@asudoh thank you for the quick turnaround on this! Is there a way to visually see this? I see that the snapshot isn't wrapping the card with the `<p>` tag, wondering what that looks like...

Reply:
@jeffchew I somehow edited https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/v1.12.0/packages/web-components/src/components/locale-modal/locale-modal-composite.ts#L151 to replace `countryList.length === 0` with `true`, where the country list comes from totally slipped from my mind... My bad.

Reply:
The snapshot looks strange... There are definitely not two `<p>` tags rendered...

Reply:
Actually it was a good catch, found that `<p>` cannot contain `<div>` according to "Permitted content" section at: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p

Pushed up a fix. Thank you for bringing this up @jeffchew!